### PR TITLE
Issue #961: Remove list image on available updates page.

### PR DESCRIPTION
**Pull request by hellomrcat**
PR for https://github.com/backdrop/backdrop-issues/issues/961

### CODE REVIEW

File: core/modules/update/update.theme.inc [hunk 1]
```diff
@@ -314,6 +314,7 @@ function theme_update_version($variables) {
     'href' => $version['release_link'],
   );
   $output .= theme('links__update_version', array('links' => $links));
+  $output = str_replace("<ul class=\"links\">", "<ul class=\"links\" style=\"list-style-type:none\">", $output);
```
Comment (docwilmot) [inline core/modules/update/update.theme.inc]: Wouldn't it be simpler to use an external CSS fix?

**Comment by hellomrcat** [comment]
@dociwlmot I'm not very familiar with using CSS, how would I go about doing that?

**Comment by hellomrcat** [comment]
@docwilmot I'm not super familiar with CSS, is this what you had in mind?

**Comment by docwilmot** [comment]
Yep, that's what I'd think would be best.

**Comment by klonos** [comment]
Was about to comment and say the same thing. Fixed :+1:

**Comment by quicksketch** [comment]
Thanks! Merged into 1.x and 1.1.x.

**Pull request merged**